yt/frontends/amrvac/tests/test_read_amrvac_namelist.py

```diff
@@ -1,13 +1,19 @@
 from copy import deepcopy
 from os import path
```

Comment:
just a note that I missed last time. it's recommended in the documentation of `path` that 

> Instead of importing this module directly, import os and refer to
this module as os.path.

Reply:
done !